src/App.tsx

```diff
@@ -1,25 +1,47 @@
 import React from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { useState } from 'react';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const lastPage = 42;
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [valuesPage, setValuesPage] = useState(5);
+
```

Comment:
It should depens on items
```suggestion

export const App: React.FC = () => {
  const [currentPage, setCurrentPage] = useState(1);
  const [valuesPage, setValuesPage] = useState(5);
  const lastPage = items[items.lenght - 1];

```